api/types/user.go

```diff
@@ -423,6 +437,24 @@ func (u UserV2) GetGCPServiceAccounts() []string {
 	return u.getTrait(constants.TraitGCPServiceAccounts)
 }
 
+// GetHostUserUID retrieves the host user uid
+func (u UserV2) GetHostUserUID() string {
+	uid := u.getTrait(constants.TraitHostUserUID)
+	if len(uid) == 0 {
+		return ""
+	}
+	return uid[0]
```

Comment:
Is there a protection mechanism somewhere that prevents us from configuring multiple UIDs per user?

Reply:
Not really, if the user manually edits the user with tctl they could specify multiple UIDs but the first one will only ever be used

Reply:
I think this should be either enforced to one UID per user, or at least documented that we take the first value. Otherwise, it's not immediately obvious what would happen if multiple ones are configured.